### ISSUE TITLE
Is there a way to see the BERT version number from R ?

### ISSUE THREAD

**Issue by AndrewRGibson** [sdllc/Basic-Excel-R-Toolkit#57]
Not a critical thing but it would be nice to include a check when I know a minimum version number is required for specific functionality.

**Comment by duncanwerner** [comment]
Good idea.  Added in f3fc5ca3b67a6ec8c289d40af1d6a089ebd4407f and will be in the next release.

**Comment by duncanwerner** [comment]
(...leave open until release)
